### ISSUE TITLE
All the versions are visible in collection list view (Strapi V 4.8.2)

### ISSUE THREAD

**Issue by cyber-m** [notum-cz/strapi-plugin-content-versioning#75]
### Describe the bug
All the versions are visible in collection list view.
In other words rows with "isVisibleInListView: false" are visible in the collection list view

The bug occurred after I updated to Strapi from 4.4.5 to 4.8.2 
I also reproduce the bug on a fresh Strapi 4.8.2 installation

### Steps to reproduce the behavior

1. Install the plugin on Strapi v4.8.2
2. Create collection type
3. Create new entry
4. Save as a new version
5. Back to collection list view

### Expected behavior
Only "the main" version of each entry should be visible in the collection list view
In other words only rows with "isVisibleInListView: true" should be visible in the collection list view

### System

- Node.js version: 14.19.1
- NPM version: 6.14.16
- Strapi version: 4.8.2
- Database: mySQL
- Operating system: MacOS 11.4

### Additional context

It used to work for me on Strapi v4.4.5

**Comment by jihyukbae** [comment]
I'm struggling to figure out how the flag `isVisibleInListView` was actually being used to filter rows in the list view.

Since neither Strapi core code nor Strapi docs have any references to it, I'm assuming that like `vuid`, `isVisibleInListView` is a flag that this plugin added - but the [existing references of it in this repo](https://github.com/notum-cz/strapi-plugin-content-versioning/search?q=isvisibleinlistview) only shows the variable being set, not how it affects the list view filtering.

As far as I can tell, this issue of all versions showing up in the list view occurs for Strapi v4.8.0+. While it's not obvious from the [release notes](https://github.com/strapi/strapi/releases/tag/v4.8.0) which changes caused this, it feels difficult to try to come up with a fix when it's unclear how the filtering worked in the first place.

Would appreciate any insights/guidance!

**Comment by sushmavoleti** [comment]
Waiting for this fix. Tried to fix by debugging. Is there any way to fix this?

**Comment by sushmavoleti** [comment]
I solved this problem by adding the below code in bootstrap.js 

   ![image](https://user-images.githubusercontent.com/3071349/235356933-a88a6966-6fed-4dcc-a73a-52788111ec38.png)

**Comment by cyber-m** [comment]
Hi @sushmavoleti @jihyukbae ,
I created a patch that fix this problem, because any changes inside node_moduls won't take effect when you deploy.

[@notum-cz+strapi-plugin-content-versioning+0.4.7.patch](https://github.com/notum-cz/strapi-plugin-content-versioning/files/11362031/%40notum-cz%2Bstrapi-plugin-content-versioning%2B0.4.7.patch)

**Comment by sushmavoleti** [comment]
I forgot to mention, I generated patch as well.

**Comment by cyber-m** [comment]
@jihyukbae You can remove it from the UI in the "Configure the view" section

<img width="509" alt="image" src="https://user-images.githubusercontent.com/20520652/235607373-3969359b-d4d5-47a0-920d-3f5c95b3ea02.png">

**Comment by jihyukbae** [comment]
aha, thank you!

**Comment by omikulcik** [comment]
Hi, I will now be taking over the maintenance of this plugin. We aim to fix this in the next milestone as we will ensure support for the >4.11. version of Strapi. For now it seems that it can be fixed by using the patch that @cyber-m created. Thank you for that! :)
